src/utils/wormhole.ts

```diff
@@ -620,6 +620,12 @@ const WORMHOLE_CHAINS: any = {
     },
     getExplorerBaseURL: function ({ network = "Mainnet", value, base }: ExplorerBaseURLInput) {
       if (base === "address") return this.explorer?.[network] + "/address/" + value;
+      if (base === "token") {
+        if (value.length <= 6) {
+          return this.explorer?.[network] + "/assets/native/" + btoa(value);
+        }
+        return this.explorer?.[network] + "/token/" + value;
```

Comment:
This would be for IBC tokens like [ATOM on Noble](https://www.mintscan.io/noble/assets/ibc/aWJjL0VGNDhFNkIxQTFBMTlGNDdFQ0FFQTYyRjU2NzBDMzdDMDU4MEU4NkE5RTg4NDk4QjdFMzkzRUI2RjQ5RjMzQzA=)

```suggestion
        return this.explorer?.[network] + "/assets/" + value;
```